doc/nrf/releases_and_maturity/migration/2.4.99-cs3_to_2.6.99-cs2/migration_guide_2.4.99-cs3_to_2.6.99-cs2_environment.rst

```diff
@@ -0,0 +1,331 @@
+
+      We recommend adding the nRF Util path to your environmental variables.
+
+Installing the |NCS| 2.6.99-cs2
+*******************************
+
+After you have installed the toolchain, complete the following steps to get the |NCS| v2.6.99-cs2:
+
+1. In the terminal window opened during the toolchain installation, initialize west with the revision of the |NCS| from the customer sampling by running the following command:
+
+   .. parsed-literal::
+      :class: highlight
+
+      west init -m https://github.com/nrfconnect/sdk-nrf --mr v2.6.99-cs2
+
+#. Enter the following command to clone the project repositories::
+
+      west update
+
+   Depending on your connection, this might take some time.
+
+#. Export a :ref:`Zephyr CMake package <zephyr:cmake_pkg>`.
+   This allows CMake to automatically load the boilerplate code required for building |NCS| applications::
+
+      west zephyr-export
+
+Your directory structure now looks similar to this::
+
+    ncs-lcs/work-dir
+    |___ .west
+    |___ bootloader
+    |___ modules
+    |___ nrf
+    |___ nrfxlib
+    |___ zephyr
+    |___ ...
+
+Note that there are additional folders, and that the structure might change.
+The full set of repositories and folders is defined in the manifest file.
+
+Updating the Terminal application
+*********************************
+
+To update `nRF Connect Serial Terminal`_, follow these steps:
+
+1. On your computer, open `nRF Connect for Desktop`_
+   If there is an update available, a pop up will notify you of its availability.
+#. If available, install the update from the pop up screen.
+#. Update `nRF Connect Serial Terminal`_ from the nRF Connect for Desktop application)
+
+If you are using the nRF Terminal application part of the `nRF Connect for Visual Studio Code`_ extension, open Visual Studio Code instead and ensure you are running the newest version of both the editor and the extension.
+
+Installing the nRF54H20 firmware bundle
+***************************************
+
+The nRF54H20 SoC requires the installation of a bundle ( :file:`nrf54h20_soc_binaries.zip`) containing the precompiled firmwares for the Secure Domain and System Controller:
+
+..
+   ### Update bundle link
+
+1. Download the `nRF54H20 firmware bundle`_.
+   The bundle is a ZIP archive that contains the following files:
+
+   * :file:`nordic_top.suit`
+   * :file:`suit_installed_envelopes_secure_merged.hex`
+   * :file:`sysctrl.hex`
+   * :file:`urot.hex`
+
+#. Unzip the bundle in a folder of your choice, then indicate the bundle folder path to nRF Util using the following command::
+
+      nrfutil device x-provision-nrf54h --firmware *path-to_bundle_.zip_file* --serial-number *serialnumber*
+
+Set the nRF54H20 to RoT
+***********************
+
+The current nRF54H20 DK is delivered with its lifecycle state (LCS) set to ``EMPTY``.
+To correctly operate, its lifecycle state must first be set to Root of Trust (``RoT``).
```

Comment:
This statement may suggest that first thing which shall be done by user is to set LCS to RoT, but actually first action shall be to program SDFW&SCFW bundle and then change LCS to RoT.